features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/InfoBar.jsx

```diff
@@ -286,6 +286,22 @@ class InfoBar extends React.Component {
         return mail;
     }
 
+    getKeyManagers(api) {
+        let keyManagers;
+        let response;
+        if ( api.keyManagers) {
```

Comment:
Shall we remove the extra space here before the word "api.keyManagers"?

Reply:
resolved